web/src/test/java/org/mskcc/cmo/metadb/web/RestIntegrationTest.java

```diff
@@ -0,0 +1,98 @@
+package org.mskcc.cmo.metadb.web;
```

Comment:
I forgot to mention that the standard naming convention for test classes is "<class name>Test" so we should rename this class "RequestControllerTest"